### PR TITLE
test(harness): bound review campaign closure

### PR DESCRIPTION
**Pull request by trvon**
## Scope
Layer 1/4, based on #93 (`fix/controller-boundary-authority`). Keep unmerged with the stack.

- Add a test-only boundary inventory and bounded review-campaign reference policy.
- Require successful current-snapshot coverage for every mandatory partition. Missing/failed/skipped/conflicting evidence cannot imply a clean review.
- Exercise repair budgets, finding deduplication, uncertainty, final gates and quiescence with adversarial fixtures.
- Document the distinction between traceability checks and behavioral proof. No production controller or new execution authority.
- Gate the process-tree timeout fixture on child readiness while preserving the production timeout callback, avoiding a host-startup timing race.

## Validation
Signed tip `995cae6`; full local and coverage gates passed (96.38% functions): 1002 tests, 16 property tests, lint/typecheck/build, package smoke (117 files), audit (zero vulnerabilities), diff/clean checks. Only two established lint warnings remain. Bounded independent source review completed this partition without blockers.

The repaired #93 base passed actual Windows and Ubuntu CI in run 33943601869. This PR's checks run separately after opening. No merge requested.

### CODE REVIEW

File: test/helpers/review-campaign.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+export function evaluateReviewCampaign(policy: ReviewPolicy, evidence: ReviewEvidence): ReviewDecision {
+  const reasons: string[] = [];
+  if (!policy.snapshot || policy.partitions.length === 0
+    || new Set(policy.partitions).size !== policy.partitions.length
+    || policy.partitions.some((id) => !id)
+    || !Number.isSafeInteger(policy.maxCalls) || policy.maxCalls < 1
+    || !Number.isSafeInteger(policy.maxRepairBatches) || policy.maxRepairBatches < 0) {
+    return { status: "incomplete", reasons: ["invalid policy"], blockers: [] };
+  }
```
[the remark this answers is not in the document]
Comment (trvon) [reply test/helpers/review-campaign.ts:41]: Fixed in 1b310e4. Policies with maxCalls below the mandatory partition count now return invalid policy before evidence evaluation. Regression covers both an insufficient budget and the exact feasible boundary. RED 3052ba3 failed as expected; updated #95 full local/coverage gate passes 1004 tests.

File: test/helpers/review-campaign.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  if (!evidence.workersQuiesced) reasons.push("worker termination unproved");
+  const sortedBlockers = [...blockers].sort();
+  if (reasons.length) return { status: "incomplete", reasons: [...new Set(reasons)].sort(), blockers: sortedBlockers };
+  if (sortedBlockers.length) {
+    const exhausted = evidence.calls >= policy.maxCalls || evidence.repairBatches >= policy.maxRepairBatches;
+    return { status: exhausted ? "incomplete" : "repair", reasons: exhausted ? ["repair budget exhausted"] : [], blockers: sortedBlockers };
+  }
+  if (!evidence.gatesPassed) reasons.push("final gates missing");
+  if (!evidence.closurePassed) reasons.push("final closure missing");
+  return { status: reasons.length ? "incomplete" : "clean", reasons, blockers: [] };
```
[the remark this answers is not in the document]
Comment (trvon) [reply test/helpers/review-campaign.ts:76]: Normalized in 1b310e4: final reasons now use the same sorted/deduplicated form as the earlier incomplete path. The previous output was deterministic, but its ordering was inconsistent. Regression checks both missing final gates and missing closure together; RED 3052ba3 then GREEN. Updated #95 gate passes 1004 tests.

File: test/helpers/review-campaign.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (trvon) [reply test/helpers/review-campaign.ts:41]: The original stack merged during validation. Preserved this fix as 31c0bfe in follow-up #100 against the integration branch. Exact follow-up head be506a6 now passes both Windows and Ubuntu CI (run 33946451270), plus 1042 local tests and 96.52% function coverage.

File: test/helpers/review-campaign.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (trvon) [reply test/helpers/review-campaign.ts:76]: Follow-up #100 preserves this change as 31c0bfe after the original stack merged. Both Windows and Ubuntu CI pass on exact head be506a6 (run 33946451270).
